### PR TITLE
Feature/responsiveness

### LINKED ISSUE #37: Refactor CSS with SASS
- [ ] variables for colors, fonts, etc.
- [ ] nesting, when/where appropriate
- [ ] at least two mixins or extends

### LINKED ISSUE #59: Display User Data

### PR DESCRIPTION
**Pull request by josharagon**
## Pull Request

<br>

### Description

* Is this a feature or a fix?
 FEATURE

* Where should the reviewer start?
INDEX.HTML AND .SCSS FILES

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
 Changes the look on smaller screen size to fit mobile and tablets.

* What outside features or research did you use to implement this fix?
NA

* How should this be tested?
Open index.html then open mobile simulator with the dev tools.
<br>

***

#### Please review considerations below:

- [ ] Follows Turing Style Guidelines

- [ ] Changes generate no new warnings or errors

- [ ] Checked code for and corrected any spelling errors

- [ ] README has been updated(if applicable)

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -171,53 +138,38 @@ h2{
   flex-direction: column;
   justify-content: space-between;
   width: 80%;
+  &-hydration {
```
Comment (PaigeVannelli) [inline src/css/style.scss:141]: Good use of SASS!

File: src/css/style.scss [hunk 2]
```diff
@@ -607,4 +553,27 @@ nav {
 
 .dropdown-menus {
   padding-left: 35px;
+}
+
+
+@media only screen and (max-width: 900px) {
```
Comment (PaigeVannelli) [inline src/css/style.scss:559]: Nice work on the media query! Do we have something for tablet or will our regular layout work?

**Review by PaigeVannelli — APPROVED** [review]
Looks good! Let's take a look at tablet media query as well

**Review by percworld — COMMENTED** [review]
@media queries - thank you.

**Pull request merged**